segments/cpu_temp.sh

```diff
@@ -0,0 +1,31 @@
+# shellcheck shell=bash
+# Prints CPU temperature
+# Requirements:
+#   lm_sensors (Linux only)
+#   smctemp (Macos only)
+
+TMUX_POWERLINE_SEG_CPU_TEMP_ICON="${TMUX_POWERLINE_SEG_CPU_TEMP_ICON:- }"
+TMUX_POWERLINE_SEG_CPU_TEMP_SENSORS_LINE_MARKER="${TMUX_POWERLINE_SEG_CPU_TEMP_SENSORS_LINE_MARKER:-Package id 0\|Physical id 0\|temp1}"
```

Comment:
The current default value, is this one grep pattern that works for  `smctemp` or `sensors`?


Could there be 2 default values set depending on `tp_shell_is_linux()` or similar?

Reply:
Only for 'sensors'.